packages/openapi-typescript/package.json

```diff
@@ -75,6 +75,7 @@
     "del-cli": "^5.1.0",
     "esbuild": "^0.20.1",
     "execa": "^7.2.0",
+    "scule": "^1.3.0",
```

Comment:
I’m fine with adding this since it’s tiny, well-documented, and well-written. But this will need to be in `dependencies` if it’s required for runtime! Otherwise it won’t install, and users will get a “module not found” error